lmfdb/elliptic_curves/elliptic_curve.py

```diff
@@ -717,20 +720,40 @@ def __init__(self):
             label="Semistable",
             example="Yes",
             knowl="ec.semistable")
+        cm_opts = []
+        for t in cm_j_invariants_and_orders(QQ):
+            el = (str(t[0]*t[1]**2), t[0]*t[1]**2)
+            if not el in cm_opts:
```

Comment:
As a Python note, you can also write this as `el not in cm_opts` using Python's `not in` operator, which reads a bit more naturally.